reference/lexical_analysis.po

```diff
@@ -1011,22 +1374,28 @@ msgid ""
 "nonzero real part, add a floating point number to it, e.g., ``(3+4j)``.  "
 "Some examples of imaginary literals::"
 msgstr ""
+"Un littéral imaginaire produit un nombre complexe dont la partie réelle est "
+"0.0. Les nombres complexes sont représentés comme une paire de nombres à "
```

Comment:
I would use double backticks if you're expressing a Python literal here. Or use a coma if you're expressing the value as a french literal.